ovs/lib/alba.py

```diff
@@ -277,6 +277,9 @@ def remove_cluster(alba_backend_guid):
                 raise RuntimeError('Node {0} is not reachable, ALBA Backend cannot be removed. {1}'.format(alba_node.ip, ce))
 
         # ACTUAL REMOVAL
+        alba_backend.backend.status = Backend.STATUSES.DELETING
+        alba_backend.invalidate_dynamics('live_status')
```

Comment:
It's not required to invalidate dynamics before a `save`, since the `save` clears all caches anyway.